### ISSUE TITLE
all container and data are reset after link a new device (mozilla account)

### ISSUE THREAD

**Issue by obarisk** [mozilla/multi-account-containers#2411]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

1. Enable sync extensions.
2. disable sync for "Multi-Account Containers".
3. Add a test container on the existed firefox.
4. Add a new firefox linked devices (Maybe the linked devices should on different OS or on different version of firefox)

### Actual behavior

Both existed and newly create firefox, existed container data are gone. All turns into default

### Expected behavior

1. For firefox before "sync", the containers should keep as it was.
2. The newly synced firefox, should have only default containers.

### Additional informations

No idea is this from I have firefox on different os and different version of firefox.

**Comment by thedefectiveram** [comment]
Yes this is constant battle for me. First initial sync was done, way back when. All the changes after are NOT synced up.
